lua/fishtank/fish.lua

```diff
@@ -1,23 +1,21 @@
 local colors = require('fishtank.colors')
-local constants = require('fishtank.constants')
 local luaUtils = require('fishtank.util.lua')
 local options = require('fishtank.options')
 local path = require('fishtank.path')
-local vimUtils = require('fishtank.util.vim')
 
-Fish = {
-    position = { row = 0, col = 0 },
-    text = '',
-    bufferID = nil,
-    windowID = nil,
-    travelPoints = {},
-}
+Fish = {}
 
 function Fish:new(o)
-    o = o or {}
+    o = o or {
+        position = { row = 0, col = 0 },
+        text = '',
+        bufferID = nil,
+        windowID = nil,
+        travelPoints = {},
```

Comment:
These are all bottom values, and I'm pretty sure they're all checked before use, so in my testing the plugin seemed to work normally even with this deleted